t2.cpp

```diff
@@ -0,0 +1,6 @@
+sdjfsjsdjfsd;
```

Comment:
variable 'sdjfsjsdjfsd' can be made static or moved into an anonymous namespace to enforce internal linkage [misc-use-internal-linkage,-warnings-as-errors]